assets/js/layzload-scripts.js

```diff
@@ -0,0 +1,86 @@
+class RocketLazyLoadScripts {
+	options = { passive: true };
+	attrName = 'data-rocketlazyloadscript';
```

Comment:
This attribute name follows the suggestion made here https://github.com/wp-media/wp-rocket/issues/2899#issuecomment-669311394. It's easily changeable as a property in this class.